cmd/root.go

```diff
@@ -97,18 +100,30 @@ func startGohaqd(cmd *cobra.Command, args []string) {
 		QueueUrl:        q.QueueUrl,
 		WaitTimeSeconds: aws.Int64(20),
 	}
+	sem = make(chan *sqs.Message)
+
+	for i := 0; i < parallelRequests; i++ {
+		go startConsumer(q.QueueUrl)
+	}
+
 	for {
```

Comment:
Can you put this `for loop` above the other one? 
Looks logically aligned. First `poll` then `process`.